packages/graphql-types-generator/src/angular/index.ts

```diff
@@ -54,6 +52,38 @@ function generateTypes(generator: CodeGenerator, context: LegacyCompilerContext)
   Object.values(context.fragments).forEach(operation => interfaceDeclarationForFragment(generator, operation));
 }
 
+function generateSubscriptionResponseWrapper(generator: CodeGenerator) {
+  generator.printOnNewline(`
+  export interface SubscriptionResponse<T> {
+    value: GraphQLResult<T>;
+  }`);
+  generator.printNewline();
+}
+function generateSubscriptionOperationTypes(generator: CodeGenerator, context: LegacyCompilerContext) {
+  const typeName = 'Subscription';
```

Comment:
Introducing `Subscription` type might have high probability to clash with developer's existing type with same name. Can we use` __Subscription` or `__SubscriptionContainer` instead?

Reply:
I think ff is not necessary because it is a fix and the original code does not work.

Reply:
I change the type name to `__SubscriptionContainer` to avoid conflicts